src/thinblock.cpp

```diff
@@ -1695,28 +1691,21 @@ void BuildSeededBloomFilter(CBloomFilter &filterMemPool,
             uint64_t nBlockSize = 0;
             while (mi != mempool.mapTx.get<3>().end())
             {
-                CTransaction tx = mi->GetTx();
-
-                if (!IsFinalTx(tx, nHeight, nLockTimeCutoff))
```

Comment:
does this mean that we are going to have a degradation in Xthin performances as soon as the fraction of mempool non final txns will start to increase?

Reply:
it just means the hashes will all get added to the bloom filter, but you know, the acutally performance issue wasn't here anyway, it was a couple of lines above where we didn't take a reference to the txn ...this was fixed in a later commit, so I'll just drop this commit and leave it as it was before.